<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerDisplaySSMModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerDisplaySSMModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="inputModel">
        <property name="text">
         <string>Input Model</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="PC">
        <property name="text">
         <string>Principal Component</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="ctkSliderWidget" name="pcSlider" native="true">
        <property name="decimals" stdset="0">
         <number>0</number>
        </property>
        <property name="minimum" stdset="0">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" stdset="0">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="std">
        <property name="text">
         <string>Standard deviation</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QPushButton" name="inputModelButton">
        <property name="text">
         <string>Open Model</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="modelName">
        <property name="text">
         <string>Model Name</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="ctkSliderWidget" name="stdSlider" native="true">
        <property name="decimals" stdset="0">
         <number>0</number>
        </property>
        <property name="minimum" stdset="0">
         <double>-10.000000000000000</double>
        </property>
        <property name="maximum" stdset="0">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLineEdit" name="modelNamePath"/>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QRadioButton" name="radioButtonITK">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>0</y>
           <width>116</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>ITK Model</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonVTK">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>0</y>
           <width>116</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>VTK Model</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pcSlider</sender>
   <signal>valueChanged(double)</signal>
   <receiver>qSlicerDisplaySSMModuleWidget</receiver>
   <slot>onSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stdSlider</sender>
   <signal>valueChanged(double)</signal>
   <receiver>qSlicerDisplaySSMModuleWidget</receiver>
   <slot>onSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputModelButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>qSlicerDisplaySSMModuleWidget</receiver>
   <slot>onSelectInputModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSelect()</slot>
  <slot>onSelectInputModel()</slot>
 </slots>
</ui>
